packages/loader/driver-utils/src/noOpDocumentDeltaStorageService.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { v4 as uuid } from "uuid";
+import { IDocumentDeltaStorageService, IStream } from "@fluidframework/driver-definitions";
+import { ISequencedDocumentMessage, MessageType } from "@fluidframework/protocol-definitions";
+import { Queue } from "./parallelRequests";
+
+/**
+ * Implementation of IDocumentDeltaStorageService that will always return NoOp when fetching messages
+ */
+export class NoOpDocumentDeltaStorageService implements IDocumentDeltaStorageService {
```

Comment:
I think it's beneficial to use same naming convention. You use "local" somewhere else, here it's "no op".